src/chrome/content/rules/Github-Pages.xml

```diff
@@ -15,11 +15,6 @@
 		<test url="http://duckietv.github.io/DuckieTV" />
 		<test url="http://duckietv.github.io/DuckieTV/" />
 
-	<!-- See https://github.com/EFForg/https-everywhere/issues/2507 -->
-	<exclusion pattern="^http://googletrends\.github\.io/(GOPDebate1|iframe-scaffolder)/" />
```

Comment:
- https://googletrends.github.io/: MCB from `font.googleapis.com` (begnin)
- https://googletrends.github.io/Sandy/index.html: MCB from `libs.cartocdn.com` and `maps.google.com`.

I'm not sure we should change this for the time being.

Reply:
@Bisaloo i've added it back and only removed gopdebate1